Sources/AdminPanelProvider/Controllers/LoginController.swift

```diff
@@ -9,7 +9,7 @@ public final class LoginController {
     }
 
     public func landing(req: Request) throws -> ResponseRepresentable {
-        return try renderer.make("Login/index", for: req)
+        return try renderer.make("Login/index", ["collapse": "true"], for: req)
```

Comment:
Should we define these design-specific values as constants in the package somewhere?